Resources/doc/cookbook/recipe_custom_action.rst

```diff
@@ -92,7 +92,10 @@ to implement a ``clone`` action.
 
     class CRUDController extends Controller
     {
-        public function cloneAction()
+        /**
+         * @param $id
+         */
+        public function cloneAction($id)
```

Comment:
I was thinking about `$this->request->get($admin->getIdParameter())`, but it's too complicated after all, drop it.